### PR TITLE
[RHICOMPL-1014] Fix policy name and type on Report pages

### PR DESCRIPTION
**Pull request by vkrizan**
Fixes Reports table and details to show policy type and user defined policy name:
* on reports table
* on report cards
* on policy popover
* on policy details (title and breadcrumbs)

Additional fixes to #787.

Screenshot_2020-11-11 Compliance
Screenshot_2020-11-11 Compliance(1)
Screenshot_2020-11-11 Compliance(2)

### CODE REVIEW

File: src/PresentationalComponents/ReportsTable/Cells.js [hunk 1]
```diff
@@ -19,11 +19,14 @@ export const Name = (profile) => (
...
-        </Label> }
-        { profile.policy && <GreySmallText>{ profile.name }</GreySmallText> }
+        { profile.policy
+            ? <React.Fragment>
+                <PolicyPopover { ...{ profile, position: 'right' } } />
+                <GreySmallText>{ profile.policyType }</GreySmallText>
+            </React.Fragment>
+
+            : <Label color="red">External</Label>
+        }
```
Comment (akofink) [inline src/PresentationalComponents/ReportsTable/Cells.js:29]: The table looks good, but the cards are still incorrect:

image

image

Comment (vkrizan) [reply src/PresentationalComponents/ReportsTable/Cells.js:29]: Isn't it out of scope? We're going to enable the table feature within its deployment or shortly after.

Comment (akofink) [reply src/PresentationalComponents/ReportsTable/Cells.js:29]: Are we? I thought it was still an open question with PM that we will resolve after our demo.

**Comment by akofink** [comment]
I'm also still seeing the policy type instead of the policy name in the policies table (both policies below have custom names that are not shown):

image

**Comment by vkrizan** [comment]
> I'm also still seeing the policy type instead of the policy name in the policies table (both policies below have custom names that are not shown):

@akofink It's a profile name to be exact. This only fixes Report pages. We'll fix it separately.

**Comment by akofink** [comment]
Oh, gotcha! I thought it was policy table as well from the PR description
> on policy table

**Comment by vkrizan** [comment]
Sorry, my bad, it supposed to be "on reports table".

File: src/PresentationalComponents/ReportsTable/Cells.js (hunk 1 above)
Comment (vkrizan) [reply src/PresentationalComponents/ReportsTable/Cells.js:29]: Fixed:
Screenshot_2020-11-12 Compliance

**Review by akofink — APPROVED** [review]
Awesome, thanks @vkrizan! I guess we are going to disable the cards soon after all.

**Pull request merged**